app/javascript/packs/application.js

```diff
@@ -4,14 +4,18 @@
 // that code so it'll be compiled.
 //= require jquery3
 //= require popper
+//= require jquery_ujs
```

Comment:
I personally don't recommend using [jQuery](https://platzi.com/blog/debes-dejar-de-usar-jquery/). Long ago it was recommended because JS was still very simple, but with the capabilities that JS currently has, it is no longer essential to compensate for it with another library.

Reply:
Deleted jQuery.